heron/stmgr/src/cpp/util/tuple-cache.cpp

```diff
@@ -91,11 +98,20 @@ TupleCache::TupleList* TupleCache::get(sp_int32 _task_id) {
   return l;
 }
 
+void TupleCache::clear() {
```

Comment:
Can you clarify in comment what the difference is between draining and clearing?